midi_app_controller/state/state_manager.py

```diff
@@ -7,7 +7,7 @@
 from app_model.types import CommandRule, MenuItem
 
 # TODO: This will be made public in some future napari version
-from napari._app_model import get_app
+from napari._app_model import get_app_model
```

Comment:
If we use a try-except here, we will be compatible with older versions of napari, before the deprecation:

```suggestion
try:
    from napari._app_model import get_app_model
except ImportError:
    from napari._app_model import get_app as get_app_model
```

@Czaki what do you think? Then I think we can just merge this and release?

Reply:
I prefer to use packaging and 

```python
from packaging.version import parse as parse_version 
from importlib.metadata import version

if parse_version(version('napari')) >= parse_version('0.5.4'):
    from napari._app_model import get_app_model
else:
    from napari._app_model import get_app as get_app_model
```

It allows for simpler found code that may be dropped when dropping support of older napari releases.